### PR TITLE
Add MBS 0867 and 0817

### PR DESCRIPTION
**Pull request by nealedj**
### What is the context of this PR?
New MBS questionnaires.

### How to review 
Check against schema definitions in slides

### CODE REVIEW

**Comment by nealedj** [comment]
Do not merge until SDX have added the new qcodes.

**Review by AndrewTorrance — CHANGES_REQUESTED** [review]
Survey id should be 009 not mbs to get correct data for eq

File: data/en/mbs_0817.json [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+                        "title": "Reporting period"
+                    },
+                    {
+                        "type": "Question",
+                        "id": "reporting-period-block-2",
+                        "title": "Reporting period",
+                        "questions": [{
+                            "type": "DateRange",
+                            "id": "reporting-period-question-2",
+                            "title": "What are the dates of the period that you will be reporting for?",
```
Comment (MIslam92) [inline data/en/mbs_0817.json:139]: There is no validation here when the user enters a period range larger then 50 days.

File: data/en/mbs_0817.json [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+                                "id": "changes-in-turnover-answer",
+                                "label": "",
+                                "description": "",
+                                "type": "Radio",
+                                "q_code": "146",
+                                "mandatory": false
+                            }],
+                            "guidance": {
+                                "content": [{
+                                    "title": "Include",
```
Comment (MIslam92) [inline data/en/mbs_0817.json]: Forgot the colon after the include here

**Review by MIslam92 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0817.json (hunk 1 above)
Comment (shahi645) [reply data/en/mbs_0817.json:139]: part of my PR then can be added

File: data/en/mbs_0867.json [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+                                "label": "Total number of employees",
+                                "mandatory": false,
+                                "q_code": "50",
+                                "type": "Number",
+                                "default": 0
+                            }],
+                            "description": "An employee is anyone aged 16 years or over that your organisation directly pays from its payroll(s), in return for carrying out a full-time or part-time job or being on a training scheme.",
+                            "guidance": {
+                                "content": [{
+                                        "title": "Include",
```
Comment (MIslam92) [inline data/en/mbs_0867.json]: Same as above forgot the colon here as well

File: data/en/mbs_0867.json [hunk 4]
```diff
@@ -0,0 +1,628 @@
...
+                                    "q_code": "54",
+                                    "type": "Number",
+                                    "max_value": {
+                                        "answer_id": "number-of-employees-total"
+                                    }
+                                }
+                            ],
+                            "guidance": {
+                                "content": [{
+                                        "title": "Include",
```
Comment (MIslam92) [inline data/en/mbs_0867.json:416]: Missing colon

File: data/en/mbs_0867.json [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+                                "id": "changes-in-turnover-answer",
+                                "label": "",
+                                "description": "",
+                                "type": "Radio",
+                                "q_code": "146",
+                                "mandatory": false
+                            }],
+                            "guidance": {
+                                "content": [{
+                                    "title": "Include",
```
Comment (MIslam92) [inline data/en/mbs_0867.json]: Missing colon

**Review by MIslam92 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0867.json [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+                    {
+                        "id": "turnover-block",
+                        "type": "Question",
+                        "questions": [{
+                            "guidance": {
+                                "content": [{
+                                        "description": "",
+                                        "title": "Include:",
+                                        "list": [
+                                            "income from sub-contracted activities",
```
Comment (MIslam92) [inline data/en/mbs_0867.json]: Missing 'exports' and 'costs incurred and passed on to customers' at the top of this Include list

**Review by MIslam92 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0867.json [hunk 7]
```diff
@@ -0,0 +1,628 @@
...
+                            "id": "number-of-employees-split-question",
+                            "title": "Of the <em>{{answers['number-of-employees-total']}}</em> total employees employed on {{exercise.start_date|format_date}}, how many male and female employees worked the following hours?",
+                            "type": "Calculated"
+                        }]
+                    },
+                    {
+                        "id": "changes-in-turnover-block",
+                        "type": "Question",
+                        "questions": [{
+                            "title": "Did any significant changes occur to the turnover for {{respondent.trad_as_or_ru_name}}?",
```
Comment (MIslam92) [inline data/en/mbs_0867.json:443]: Radio options are not mandatory here

**Review by MIslam92 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0867.json (hunk 3 above)
Comment (nealedj) [reply data/en/mbs_0867.json]: Good catch!

File: data/en/mbs_0817.json [hunk 8]
```diff
@@ -0,0 +1,390 @@
...
+                                        "value": "No",
+                                        "label": "No"
+                                    }
+                                ],
+                                "id": "changes-in-turnover-answer",
+                                "label": "",
+                                "description": "",
+                                "type": "Radio",
+                                "q_code": "146",
+                                "mandatory": false
```
Comment (shahi645) [inline data/en/mbs_0817.json]: needs to be mandatory

**Review by shahi645 — CHANGES_REQUESTED** [review]
small change

**Comment by charlotte-hirst** [comment]
Once checkboxes made mandatory i am happy with both forms

**Review by shahi645 — APPROVED** [review]
LGTM

File: data/en/mbs_0817.json [hunk 9]
```diff
@@ -0,0 +1,390 @@
...
+                                    "The information you provide contributes to <a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/economy/grossdomesticproductgdp'>Gross Domestic Product (GDP).</a>"
+                                ]
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "reporting-period-block",
+                        "type": "Question",
+                        "questions": [{
+                            "description": "",
```
Comment (MIslam92) [inline data/en/mbs_0817.json]: If were not using these we should remove all instances of the description that we are not using and labels too.

File: data/en/mbs_0817.json [hunk 10]
```diff
@@ -0,0 +1,390 @@
...
+                                "label": "Total turnover excluding VAT",
+                                "description": "",
+                                "type": "Currency",
+                                "currency": "GBP",
+                                "decimal_places": 2,
+                                "q_code": "40",
+                                "mandatory": true
+                            }],
+                            "type": "General",
+                            "title": "For the period {{ format_conditional_date (answers['reporting-period-answer-from'], exercise.start_date)}} to {{ format_conditional_date (answers['reporting-period-answer-to'], exercise.end_date)}}, what was the value of {{respondent.trad_as_or_ru_name}}’s <em>total turnover</em>, excluding VAT?"
```
Comment (davecharles) [inline data/en/mbs_0817.json]: Be consistent with `{{val` rather than `{{ val`

File: data/en/mbs_0817.json [hunk 11]
```diff
@@ -0,0 +1,390 @@
...
+                        }],
+                        "title": "Turnover"
+                    },
+                    {
+                        "id": "changes-in-turnover-block",
+                        "type": "Question",
+                        "questions": [{
+                            "title": "Did any significant changes occur to the turnover for {{respondent.trad_as_or_ru_name}}?",
+                            "id": "changes-in-turnover-question",
+                            "description": "<p>Please note: what constitutes a ‘significant change’ is dependent on your own interpretation in relation to {{respondent.trad_as_or_ru_name}}’s figures from the previous reporting period and the same reporting period last year. </p><p>This information will help us to validate your data and should reduce the need to query any figures with you. </p>",
```
Comment (davecharles) [inline data/en/mbs_0817.json:206]: In surveys we have a mix of `‘`, `’`, `'` and even `&apos;` (lfs, my bad - I think author made me do it), as well as `”`, `"` and `\"`, not sure it matters but just thought I'd point out.

Basically - I suspect the unicode `‘`, `’`, `”` etc are coming from a cut and paste from another document and I think we should clear on what is _meant_ to be in the schema.

File: data/en/mbs_0867.json [hunk 12]
```diff
@@ -0,0 +1,629 @@
...
+                        "title": "Reporting period"
+                    },
+                    {
+                        "type": "Question",
+                        "id": "reporting-period-block-2",
+                        "title": "Reporting period",
+                        "questions": [{
+                            "type": "DateRange",
+                            "id": "reporting-period-question-2",
+                            "title": "What are the dates of the period that you will be reporting for?",
```
Comment (davecharles) [inline data/en/mbs_0867.json:193]: Date validation?

Comment (nealedj) [reply data/en/mbs_0867.json:193]: Waiting on date range validation to be merged.

File: data/en/mbs_0817.json (hunk 11 above)
Comment (nealedj) [reply data/en/mbs_0817.json:206]: Correct apostrophe should always be `‘`. Ben showed me a description as to why but I can't remember where it was.

Here's a previous fix for reference https://github.com/ONSdigital/eq-survey-runner/issues/1458

Comment (shahi645) [reply data/en/mbs_0817.json:206]: Should always be smart quotes I believe

**Review by MIslam92 — APPROVED** [review]
LGTM

File: data/en/mbs_0867.json [hunk 13]
```diff
@@ -0,0 +1,614 @@
...
+                                            "agency workers paid directly from the agency payroll",
+                                            "voluntary workers",
+                                            "former employees only receiving a pension",
+                                            "self-employed workers"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "number-of-employees-total-question",
+                            "title": "On {{exercise.start_date|format_date}}, what was the number of employees for {{respondent.trad_as_or_ru_name}}?",
```
Comment (shahi645) [inline data/en/mbs_0867.json:287]: Needs to be `employment_date` not start date

**Review by shahi645 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0867.json [hunk 14]
```diff
@@ -0,0 +1,614 @@
...
+                                    },
+                                    {
+                                        "label": "No I need to change this",
+                                        "value": "No"
+                                    }
+                                ],
+                                "mandatory": true
+                            }],
+                            "id": "confirm-zero-employees-question",
+                            "title": "On {{exercise.start_date|format_date}}, the number of employees for {{respondent.trad_as_or_ru_name}} was <em>0</em>, is this correct?"
```
Comment (shahi645) [inline data/en/mbs_0867.json:322]: employment date

File: data/en/mbs_0867.json [hunk 15]
```diff
@@ -0,0 +1,614 @@
...
+                                            "voluntary workers",
+                                            "former employees only receiving a pension",
+                                            "self-employed workers"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "number-of-employees-split-question",
+                            "title": "Of the <em>{{answers['number-of-employees-total']}}</em> total employees employed on {{exercise.start_date|format_date}}, how many male and female employees worked the following hours?",
+                            "type": "Calculated"
```
Comment (shahi645) [inline data/en/mbs_0867.json:436]: employment date

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Comment by nealedj** [comment]
All changes addressed - merging to feature branch

**Pull request merged**